src/lib.rs

```diff
@@ -359,6 +359,16 @@ const MAX_UNDECRYPTABLE_PACKETS: usize = 10;
 
 const RESERVED_VERSION_MASK: u32 = 0xfafafafa;
 
+// The default size of the receiver connection flow control window.
+const DEFAULT_CONNECTION_WINDOW: u64 = 48 * 1024;
+
+// The maximum size of the receiver connection flow control window.
+const MAX_CONNECTION_WINDOW: u64 = 24 * 1024 * 1024;
```

Comment:
We might want to make the maximum configurable by the application. But for now this is ok.

Reply:
the max window (connection, stream) is now configurable.